includes/class-wp-job-manager-usage-tracking.php

```diff
@@ -182,7 +182,9 @@ protected function get_event_prefix() {
 	}
 
 	/**
-	 * Get the text domain used in the plugin.
+	 * Get the text domain used in the plugin. Deprecated - use 'wp-job-manager' directly.
+	 *
+	 * @deprecated 1.41.1
```

Comment:
I believe we should use `$$next-version$$` here, no?

```suggestion
	 * @deprecated $$next-version$$
```